neurodamus/gap_junction.py

```diff
@@ -85,10 +85,11 @@ def _compute_gj_offsets(self, gj_dir):
         gj_offsets = compat.Vector()
         gj_sum = 0
 
-        for line in open(gjfname):
-            gj_offsets.append(gj_sum)  # fist gid has no offset. the final total is not used
-            _, offset = map(int, line.strip().split())
-            gj_sum += 2 * offset
+        with open(gjfname) as fd:
+            for line in fd:
+                gj_offsets.append(gj_sum)  # fist gid has no offset. the final total is not used
+                _, offset = map(int, line.strip().split())
```

Comment:
didn't you want to opt for `_<name>`?

Reply:
yeah, but I don't know enough about this code, I'm just fixing the dangling file pointer

Reply:
ah c'mon. I think it is safe if you put a new unused variable there with a `_` affix

Reply:
just this and you get an approval :D

Reply:
the original code was `_, offset = map(int, line.strip().split())`, so it wasn't new.
I don't know what's supposed to be in `gjinfo.txt` is the problem - it's not documented anywhere that I know of